GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1662,30 +1649,32 @@ private void ChangeWorkingDir(string path)
                 }
             }
 
-            bool removedAll = false;
-            if (invalidPaths.Count > 1)
-            {
-                dialogResult = MessageBox.Show(this, string.Format(_directoriesAreNotValidRepositoriesFormat.Text, invalidPaths.Count),
-                                                            _directoriesAreNotValidRepositoriesCaption.Text,
-                                                            MessageBoxButtons.YesNo,
-                                                            MessageBoxIcon.Exclamation,
-                                                            MessageBoxDefaultButton.Button1);
+            string dialogTitle = _directoryIsNotAValidRepositoryCaption.Text;
+            string mainInstruction = _directoryIsNotAValidRepositoryMainInstruction.Text;
+            string content = "";
```

Comment:
Please inline all above at the call site, use parameter name annotations or `/* */`, if necessary